site/documentation/content/user-guide/coap-adapter.md

```diff
@@ -0,0 +1,230 @@
+Please refer to the [Credentials API]({{< relref "/api/credentials#standard-credential-types" >}}) for details regarding the different types of secrets.
+
+**NB** There is a subtle difference between the *device identifier* (*device-id*) and the *auth-id* a device uses for authentication. See [Device Identity]({{< relref "/concepts/device-identity.md" >}}) for a discussion of the concepts.
+
+## Message Limits
+
+Before accepting any telemetry or event or command messages, the CoAP adapter verifies that the configured [message limit]({{< relref "/concepts/resource-limits#messages-limit" >}}) is not exceeded.
+For CoAP currently only the messages limit is verified, the connections limit is not available. If the limit is exceeded then the incoming message is discarded with the status code `429 Too Many Requests`. 
+
+## CoAP Content Formats
+
+CoAP doesn't use a textual identifier for content types. Instead numbers are used, which are maintained by the [IANA](https://www.iana.org/).
+For the currently assigned numbers, please check [IANA - CoAP Content Formats](https://www.iana.org/assignments/core-parameters/core-parameters.xhtml#content-formats).
+Content types that do not map to one of the registered CoAP content formats are ignored.
+
+## Publish Telemetry Data (authenticated Device)
+
+The device is authenticated using PSK.
+
+* URI: `/telemetry`
+* Method: `POST`
+* Type:
+  * `CON`: *at least once* (`1`) QoS levels
+  * `NON`: *at most once* (`0`) QoS levels
+* Request Options:
+  * (optional) `content-format`: The type of payload contained in the request body. Required, if request contains payload.
+  * (optional) `URI-query: hono-ttd`: The number of seconds the device will wait for the response.
+  * (optional) `URI-query: empty`: If the request doesn't contain payload.
+* Request Body:
+  * (optional) Arbitrary payload encoded according to the given content type. Maybe empty, if `URI-query: empty` is provided.
+* Response Options:
+  * (optional) `content-format`: A media type describing the semantics and format of payload contained in the response body. This option will only be present if the response contains a command to be executed by the device which requires input data. If the command's content type do not map to one of the registered CoAP content formats, the option will not be present.
+  * (optional) `location-query: hono-command`: The name of the command to execute. This option will only be present if the response contains a command to be executed by the device.
+  * (optional) `location-path: ('command'|'command_response/'<command-request-id>)`: The location path contains `'command'` for one-way-commands, or `'command_response' / <command-request-id>`  for commands expecting  a response. In case of a expected response, the location-path contains exactly the URI-path expected for the response.
```

Comment:
wouldn't it be sufficient to simply omit this option in case of a one-way command?

Reply:
At least for the gateway-variant below, it's required.
Using it enables later "homogeneous" extensions, otherwise "none" must be a extra case, which defaults then to "command".